lib/livebook/intellisense.ex

```diff
@@ -438,6 +440,22 @@ defmodule Livebook.Intellisense do
     """
   end
 
+  defp format_hexdoc(module, name \\ nil, arity \\ nil) do
+    fragment =
+      cond do
+        is_nil(name) -> ""
+        is_nil(arity) -> "##{name}"
+        true -> "##{name}/#{arity}"
+      end
+
+    url =
+      "https://hexdocs.pm/elixir/#{System.version()}/#{inspect(module)}.html"
+      |> URI.parse()
+      |> URI.merge(fragment)
+
+    "[hexdocs.pm](#{url})"
```

Comment:
We actually need to fetch the application for module:

```suggestion
    app = Application.get_application(module)
    
    if vsn = app && Application.spec(app, :vsn) do
      url = "https://hexdocs.pm/#{app}/#{vsn}/#{inspect(module)}.html#{fragment}"
      "[hexdocs.pm](#{url})"
    end
```

With this, for example, you should also be able to highlight modules coming from dependencies!